### PR TITLE
Updated readme to match latest changes in main score spec repo

### PR DESCRIPTION
**Pull request by sujaya-sys**
Updated readme to match latest changes in https://github.com/score-spec/spec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +95,10 @@ Flags:
       --patch-manifests stringArray     An optional set of <kind|*>/<name|*>/path=key operations for the output manifests
 ```
 
+## Installation
+
+To install `score-k8s`, follow the instructions as described in our [installation guide](https://docs.score.dev/docs/score-implementation/score-k8s/#installation).
```
Comment (astromechza) [inline README.md:100]: @sujaya-sys these repos' should be self-contained to allow for forking the project correctly. I suggest we don't deduplicate this content.

File: README.md [hunk 2]
```diff
@@ -223,3 +164,29 @@ $ score-k8s resources get-outputs 'dns.default#demo-app.dns' --format '{{.host}}
...
 
 Resource cleanup has not been implemented yet. The only mechanism today is limited to deleting the Kubernetes manifests output by a template provisioner. As a workaround, the YAML structure in `.score-k8s/state.yaml` can be interpreted to determine what side effects need to be cleaned up.
+
+## Get in touch
+
+Connect with us through the [Score Slack channel](https://join.slack.com/t/scorecommunity/shared_invite/zt-2a0x563j7-i1vZOK2Yg2o4TwCM1irIuA) or contact us via email at team@score.dev.
+
+We host regular community meetings to discuss updates, share ideas, and collaborate. Here are the details:
+
+| Community call | Info |
```
Comment (astromechza) [inline README.md:174]: Don't duplicate this content here. Instead indicate that this information can be found in the main score readme.

File: README.md (hunk 1 above)
Comment (astromechza) [reply README.md:100]: espescially the install from source

**Review by astromechza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sujaya-sys) [reply README.md]: good point!

**Review by astromechza — APPROVED** [review]

**Pull request merged**